website/bot.py

```
@@ -1,202 +1,182 @@
+            result = chain.invoke({"input": safe_message})
+            answer = result.get("answer") or result.get("output") or "Sorry, I couldn’t find an answer."
+
+            cache.set(safe_message, answer, timeout=3600)
+
+            log_chat(f"[INPUT] {safe_message}")
+            log_chat(f"[OUTPUT] {answer}")
+            return Response({"message": answer}, status=status.HTTP_200_OK)
+
+            
+        except Exception as e:
+            log_chat(f"[ERROR] {str(e)}")
```

Ruff (0.14.7)

Use explicit conversion flag

Replace with conversion flag

(RUF010)